adb-server/adbserver-desktop/src/main/java/com/kaspersky/adbserver/desktop/Desktop.kt

```diff
@@ -17,17 +20,35 @@ internal class Desktop(
     }
 
     private val devices: MutableCollection<DeviceMirror> = mutableListOf()
+    private var isRunning = AtomicBoolean(false)
 
-    fun startDevicesObserving() {
+    fun startDevicesObservingSync() {
```

Comment:
nit: I would just mention that this method is blocking in documentation as you did in AdbCommandPerformer::perform. I think blocking behavior is an expected one. Another hint is that there's startDevicesObservingAsync

Reply:
yes, I'm add documentation to this methods